File: Bioinformatics Stronghold/DNA - Counting DNA Nucleotides/DNA.py
#Importing libraries
import pyperclip

#Reading variables txt file
file = open('rosalind_dna.txt', 'r')
dna_string = file.read().strip()

#Counting
nucleotide_counter = {}
for base in dna_string:
	if base in nucleotide_counter:
		nucleotide_counter[base] += 1
	else:
		nucleotide_counter[base] = 1

#Printing accordingly
print(nucleotide_counter['A'], nucleotide_counter['C'], nucleotide_counter['G'], nucleotide_counter['T'])

#Copying to clipboard for easier submition
pyperclip.copy(nucleotide_counter['A'], nucleotide_counter['C'], nucleotide_counter['G'], nucleotide_counter['T'])

File: Bioinformatics Stronghold/DNA - Counting DNA Nucleotides/DNA.rb
#Reading data from file
readData = File.open("rosalind_dna.txt")
dna_string = readData.read

#Counting
a_count=0
t_count=0
c_count=0
g_count=0

for i in 0..dna_string.length()
	base = dna_string[i]
	if (base != '')
		if (base == 'A')
			a_count += 1
		elsif (base == 'T')
			t_count += 1
		elsif (base == 'C')
			c_count += 1
		elsif (base == 'G')
			g_count += 1
		end
	end
end

#Printing results
print "#{a_count} #{c_count} #{g_count} #{t_count}"